prestoadmin/prestoclient.py

```diff
@@ -191,3 +219,91 @@ def get_rows(self, num_of_rows=NUM_ROWS):
 
     def get_next_uri(self):
         return self.next_uri
+
+    def _get_connection(self):
+        if self.coordinator_config.use_https():
+            return self._get_https_connection()
+        else:
+            return HTTPConnection(self.server, self.port, False, URL_TIMEOUT_MS)
+
+    def requires_https(self):
+        return self.coordinator_config.use_https()
+
+    def _get_https_connection(self):
+        ca_file_path = self.get_pem(env.conf)
+        https_port = self.coordinator_config.get_https_port()
+        result = HTTPSCaCertConnection(
+                self.server, https_port, None, None, ca_file_path, False, URL_TIMEOUT_MS)
+        return result
+
+    def _get_local_keystore_path(self):
+        remote_keystore_path = self.coordinator_config.get_client_keystore_path()
+        keystore_name = os.path.basename(remote_keystore_path)
+        return os.path.join(get_coordinator_directory(), keystore_name)
```

Comment:
This seems like it's assuming that PA is installed on the same host as the Presto coordinator (unless get_coordinator_directory() is the config directory)